launch/start.launch

```diff
@@ -0,0 +1,39 @@
+<launch>
+  
+  <node pkg="polhemus" type="polhemus_tf_broadcaster" name="polhemus_tf_broadcaster"/>
+
+  <!--Set output hemisphere parameters-->
+  <param name="x_hs" value="0" type="double"/>
+  <param name="y_hs" value="0" type="double"/>
+  <param name="z_hs" value="-1" type="double"/>
+
+  <!--Setup static transform polhemus_base <-> world-->
+  <node pkg="tf" type="static_transform_publisher" name="polhemus_world_broadcaster" args="0 0 0.54 0 0 3.1415926 polhemus_world polhemus_base 100"/>
+
+  <!--Specify attached stations and those to broadcast-->
+  <param name="station_1_attached" value="true" type="bool"/>
+  <param name="station_1_broadcast" value="true" type="bool"/>
+
+  <param name="station_2_attached" value="true" type="bool"/>
+  <param name="station_2_broadcast" value="false" type="bool"/>
+
+  <param name="station_3_attached" value="true" type="bool"/>
+  <param name="station_3_broadcast" value="false" type="bool"/>
+
+  <param name="station_4_attached" value="true" type="bool"/>
+  <param name="station_4_broadcast" value="false" type="bool"/>
+
+  <!--These are for polhemus liberty with 8 stations-->
+  <param name="station_5_attached" value="false" type="bool"/>
+  <param name="station_5_broadcast" value="false" type="bool"/>
+
+  <param name="station_6_attached" value="false" type="bool"/>
+  <param name="station_6_broadcast" value="false" type="bool"/>
+
+  <param name="station_7_attached" value="false" type="bool"/>
+  <param name="station_7_broadcast" value="false" type="bool"/>
+
+  <param name="station_8_attached" value="false" type="bool"/>
+  <param name="station_8_broadcast" value="false" type="bool"/>
+
```

Comment:
Having just one flag for not/connected should be enough. Additional frames can just be ignored. Remove the broadcast flag and assume its always true.